code/game/gamemodes/clock_cult/clock_structures/traps/brass_skewer.dm

```diff
@@ -49,20 +51,31 @@
 		playsound(squirrel, 'sound/effects/splat.ogg', 50, TRUE)
 		playsound(squirrel, 'sound/misc/desceration-03.ogg', 50, TRUE)
 		squirrel.apply_damage(20, BRUTE, "chest")
-		squirrel.pixel_y = 8
+		mouse_opacity = MOUSE_OPACITY_OPAQUE //So players can interact with the tile it's on to pull them off
 		buckle_mob(squirrel, TRUE)
 	else
 		visible_message("<span class='danger'>A massive brass spike erupts from the ground!</span>")
 	playsound(src, 'sound/machines/clockcult/brass_skewer.ogg', 75, FALSE)
 	icon_state = "[initial(icon_state)]_extended"
 	density = TRUE //Skewers are one-use only
-	desc = "A massive brass spike protruding from the ground like a snapped bone. It makes you sick to look at."
+	desc = "A vicious brass spike protruding from the ground like a stala[pick("gm", "ct")]ite. It makes you sick to look at." //is stalagmite the ground one? or the ceiling one? who can ever remember?
 
 /obj/structure/destructible/clockwork/trap/brass_skewer/user_buckle_mob()
 	return
 
+/obj/structure/destructible/clockwork/trap/brass_skewer/post_buckle_mob(mob/living/L)
+	if(L in buckled_mobs)
+		L.pixel_y = 3
+		impale_overlay = mutable_appearance('icons/obj/clockwork_objects.dmi', "brass_skewer_pokeybit", ABOVE_MOB_LAYER)
+		L.add_overlay(impale_overlay)
```

Comment:
it'd be much better for this to be an overlay on the skewer itself rather than the mob, and I don't see any reason why it can't be.